automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/SignInSpec.scala

```diff
@@ -13,14 +14,27 @@ class SignInSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Mat
 
   "A user with a registered account" - {
 
-      "should be able to log in and out" in {
-        // "harry": "harry.potter@quality.firecloud.org"
-        val student = Config.Users.testUser
-        withWebDriver { implicit driver =>
-          withSignInReal(student) { listPage =>
-            eventually { listPage.readUserEmail() shouldEqual student.email }
-          }
+    "should be able to log in and out as harry.potter" in {
+      // config is harded to get "harry": "harry.potter@quality.firecloud.org"
+      val harry = Config.Users.testUser
```

Comment:
let's not call him "harry" here, since that value can be set in the application.conf so it's not necessarily hard-coded to harry.  let's just say 'signinUser1' or something